### ISSUE TITLE
💳 Stripe Integration

### ISSUE THREAD

**Issue by gogl92** [riasenpai/Buymeacoffee-clone#3]
This project is really cool, adding Stripe as a payment provider option could help to be used in other countries.

**Comment by riasenpai** [comment]
Yeah that will be a good feature. Actually I made this project just to practice payment integration method.
